site/cds_rdm/inspire_harvester/jobs.py

```diff
@@ -56,24 +57,34 @@ class InspireArgsSchema(Schema):
     @validates_schema
     def validate_date_range(self, data, **kwargs):
         """Ensure that since <= until."""
+        current_app.logger.debug("Start validating date range.")
+
         since = data.get("since")
         until = data.get("until")
 
         if since and until and since > until:
+            current_app.logger.error(
+                f"Validation failed. 'Since' value: {since}, 'Until' value: {until}. See "
+                "ValidationError message."
+            )
             raise ValidationError(
                 _("The 'Since' date must be earlier than or equal to the 'Until' date.")
             )
+        current_app.logger.debug(f"Successfully validated the date range.")
 
     @validates_schema
     def validate_exclusive_arguments(self, data, **kwargs):
         """Ensures that the user provides valid combinations of parameters."""
+        current_app.logger.debug("Start validating job parameters.")
+
         inspire_id = data.get("inspire_id")
         on_date = data.get("on_date")
         since = data.get("since")
         until = data.get("until")
 
         # if `inspire_id` is provided, no other fields should be present
         if inspire_id and any([on_date, until]):
+            current_app.logger.error("Validation failed. See ValidationError message.")
```

Comment:
I think these are too much alongside the validation errors, because we will not even start running the task if the validation fails. The log will never show up in the UI, but user will know something is wrong form the validation errors in the form